### PR TITLE
CASSANDRA-16715 Updated website with content for June 2021

### PR DESCRIPTION
**Pull request by ErickRamirezAU**
The June 2021 website updates include:
- Homepage - Added Changelog 7 and K8s SIG update 2, Instagram & Hornet logos
- Blog - Added 2 new posts for monthly Changelog and K8s SIG
- Case Studies - Added Hornet
- Ecosystem - Added 40 listings

### CODE REVIEW

File: content/index.html [hunk 1]
```diff
@@ -343,7 +343,11 @@ <h2 class="white py-small">Get started with Cassandra, fast.</h2>
...
+>>>>>>> CASSANDRA-16715 Updated website with content for June 2021
```
Comment (michaelsembwever) [inline content/index.html]: @ErickRamirezAU this needs to be resolved^

Comment (ErickRamirezAU) [reply content/index.html]: @michaelsembwever there were 2 changes to that list -- added `hornet.png` (with new image included in `img/companies/`) and replaced `instagram.jpg` with `instagram.png` (also in `img/companies/`). Is that what you meant?

Comment (ErickRamirezAU) [reply content/index.html]: Fixed with 02979d4.

**Comment by michaelsembwever** [comment]
staged with https://github.com/apache/cassandra-website/commit/85e58d6163b8e592309ebff2632c75d611edac7a
 https://cassandra.staged.apache.org/

**Comment by michaelsembwever** [comment]
staging updated.

**Comment by michaelsembwever** [comment]
staging updated.

**Comment by michaelsembwever** [comment]
live with 3b95a5e

**Pull request closed without merging**